test/tests_for_sdkmr/EE2Runjob_test.py

```diff
@@ -812,6 +831,7 @@ def _check_common_mock_calls_batch(mocks, reqs1, reqs2, parent_wsid, wsid):
         app=_APP_1,
         git_commit=_GIT_COMMIT_1,
         source_ws_objects=[_WS_REF_1, _WS_REF_2],
+        wsid=parent_wsid,
```

Comment:
I think we can remove the regular wsid argument from the method sig now, right?

Reply:
From _create_job ? I don't think so, it's required to build up a job to compare to.

Reply:
No, from `_check_common_mock_calls_batch` since all the wsids should be the parent wsid

Reply:
:+1: